src/reducers/lookup.js

```diff
@@ -0,0 +1,32 @@
+const DEFAULT_STATE = {
+  formData: []
+}
+
+const changeMyItems = (state, action) => {
+  let newFormData = state.formData.slice(0)
+  let needNewItemArray = true;
+
+  for (let field of newFormData) {
+    if (field.id == action.payload.id) {
+      field.items = action.payload.items
+      needNewItemArray = false;
+      break;
+    }
+  }
+
+  if (needNewItemArray) {
+    newFormData.push(action.payload)
+  }
+  return {formData: newFormData}
+}
+
+const lookup = (state=DEFAULT_STATE, action) => {
+  switch(action.type) {
+    case 'CHANGE_SELECTIONS':
```

Comment:
This should be a separate action from the `list` reducer's CHANGE_SELECTIONS. Looking at global state created by the CombineReducer, both the `list` and `lookup` namespaces in Redux are identical.

Reply:
Changed it to use just one `lookups` combined reducer for both InputList and InputLookup components.